pkg/apis/metal/types_cloudprofile.go

```diff
@@ -19,12 +19,6 @@ type CloudProfileConfig struct {
 	MachineImages []MachineImages
 	// RegionConfigs is the list of supported regions.
 	RegionConfigs []RegionConfig
-	MachineTypes  []MachineType
```

Comment:
I am unsure about dropping the machine types from the CloudConfig.

Reply:
I have retained `MachineTypes`. WorkerConfig now has `extraServerLabels` that are added on top.